frontend/src/AppReceiver.tsx

```diff
@@ -0,0 +1,4 @@
+import SignInContainer from './pages/SignPages/SignInContainer';
```

Comment:
Do we really need this file? Let's remove it

Reply:
I replace it and added other files. Maybe will this file will be receiver for import? For example, path, components, other constants